tests/test_views.py

```diff
@@ -779,6 +794,52 @@ def test_manual_rebuild(self, time, from_advisory_id, publish):
             {'msg_id': 'manual_rebuild_123', u'errata_id': 1,
              'requester': 'root'})
 
+    @patch('freshmaker.messaging.publish')
+    @patch('freshmaker.parsers.internal.manual_rebuild.ErrataAdvisory.'
+           'from_advisory_id')
+    @patch('freshmaker.parsers.internal.manual_rebuild.time.time')
+    @patch('freshmaker.views._validate_rebuild_request', return_value=None)
```

Comment:
Why is this mocked?

Reply:
Because we are testing other method, not `_validate_rebuild_request`.

Reply:
Could you please create a test then that runs that function for a manual bundle rebuid?

Reply:
Added